src/main/java/Peter/Peter.java

```diff
@@ -0,0 +1,23 @@
+package peter;
+
+import utilities.CommandParser;
+import utilities.Storage;
+import utilities.TaskList;
+
+public class Peter {
+    public static final String FILE_PATH = "tasks/data.txt";
+    private Storage store;
+    private TaskList tasklist;
+
+    public Peter() {
+        this.store = new Storage(FILE_PATH);
```

Comment:
Since `store` is not ambiguous here, omitting the `this` would be okay (refer to code guidelines under "Avoid unnecessary use of this with fields.")